vdom/src/com/vdom/api/Card.java

```diff
@@ -113,5 +114,7 @@
 
     public PileCreator getPileCreator();
 
+    public Type[] getTypes();
```

Comment:
It's required for Courtier, If the card is an estate inheriting another card I need to know the types of the inherited card. (Cannot use the getNumberOfTypes method because that way the Victory Type could be counted double (by having an Estate inheriting an Action-Victory card).